eslint-rules/tests/lib/rules/component-prop-deprecation.js

```diff
@@ -97,6 +97,22 @@ ruleTester.run('component-prop-deprecation', rule, {
       options: ruleOptions,
       code: 'import {Picker} from \'module-with-deprecations\'; pickerProps = {t: "title", s:"subtitle", migrate: true}; <Picker {...pickerProps}/>'
     },
+    {
+      options: ruleOptions,
+      code: `
+import {Picker} from 'module-with-deprecations';
+const getPickerProps = () => ({t: "title", s:"subtitle", migrate: true});
+<Picker {...getPickerProps()}/>
+`
+    },
+    {
+      options: ruleOptions,
+      code: `
+import {Picker} from 'module-with-deprecations';
+pickerProps = {t: "title", s:"subtitle", migrate: true};
+<Picker {...this.pickerProps}/>
+`
```

Comment:
Please fix the indentation here :)

Reply:
Updated automatically, don't think it fixed what you meant, but I'm not really sure which line you wanted me to fix :/

Reply:
lines 103-106 and 111-114
each line should have an indentation of 8 spaces but have 0

Reply:
I don't see any errors...
I'll assume you meant 6 spaces, I've indented the same as the rest of the file.